GTMAppAuthSwift/Sources/KeychainStore.swift

```diff
@@ -79,6 +79,8 @@ public final class KeychainStore: NSObject {
     self.itemName = itemName
     self.keychainAttributes = keychainAttributes
     self.keychainHelper = keychainHelper
+
+    NSKeyedUnarchiver.setClass(AuthState.self, forClassName: "GTMAppAuthFetcherAuthorization")
```

Comment:
Add a comment providing some context.

Reply:
Added one for unarchiving and for archiving